queries/futures/useGetAverageFundingRateForMarkets.ts

```diff
@@ -34,7 +34,7 @@ const useGetAverageFundingRateForMarkets = (
 	const { synthetixjs } = Connector.useContainer();
 	const futuresEndpoint = getFuturesEndpoint(network);
 
-	return useQueries<FundingRateResponse[]>(
+	return useQueries(
```

Comment:
was this removed for a particular reason?

or should this be reverted?

Reply:
Lint was complaining about this